packages/forms/src/lib/auto-complete/components/auto-complete/auto-complete.component.scss

```diff
@@ -1,4 +1,4 @@
-.aui-auto-complete,
-.aui-auto-complete__info {
+.o-auto-complete,
```

Comment:
wat is de redenatie om hier een organism van te maken?
als bvb een modal, range-slider of datepicker molecules zijn lijkt dit mij ook een molecule? of we moeten een hoop classes herzien